src/index.js

```diff
@@ -7,116 +7,91 @@ const {
-    // this is a bank identifier which will be used to link bills to bank operations. These
-    // identifiers should be at least a word found in the title of a bank operation related to this
-    // bill. It is not case sensitive.
-    identifiers: ['books']
+    identifiers: ['glisshop'],
+    contentType: 'application/pdf'
   })
 }
 
-// this shows authentication using the [signin function](https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/docs/api.md#module_signin)
-// even if this in another domain here, but it works as an example
 function authenticate(username, password) {
   return signin({
-    url: `http://quotes.toscrape.com/login`,
-    formSelector: 'form',
-    formData: { username, password },
-    // the validate function will check if
+    url: `${baseUrl}/connexion/`,
+    formSelector: 'form[role="form"]',
+    formData: {
+      'account[mail]': username,
+      'account[pass]': password
+    },
     validate: (statusCode, $) => {
-      // The login in toscrape.com always works excepted when no password is set
-      if ($(`a[href='/logout']`).length === 1) {
+      if (
+        $(`a[href='https://www.glisshop.com/customers/logout/']`).text() ===
+        'Déconnexion'
+      ) {
         return true
       } else {
-        // cozy-konnector-libs has its own logging function which format these logs with colors in
-        // standalone and dev mode and as JSON in production mode
         log('error', $('.error').text())
         return false
       }
     }
   })
 }
 
-// The goal of this function is to parse a html page wrapped by a cheerio instance
-// and return an array of js objects which will be saved to the cozy by saveBills (https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/docs/api.md#savebills)
 function parseDocuments($) {
-  // you can find documentation about the scrape function here :
-  // https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/docs/api.md#scrape
   const docs = scrape(
     $,
     {
-      title: {
-        sel: 'h3 a',
-        attr: 'title'
+      commandNumber: {
+        sel: 'div[class=order-label] span',
+        parse: string => string.slice(9)
       },
       amount: {
-        sel: '.price_color',
-        parse: normalizePrice
+        sel: '.order-amount',
+        // Warning : Not a standard space
+        parse: amount => parseFloat(amount.split(' ')[0])
       },
-      fileurl: {
-        sel: 'img',
-        attr: 'src',
-        parse: src => `${baseUrl}/${src}`
+      currency: {
+        sel: '.order-amount',
+        // Warning : Not a standard space
+        parse: amount => amount.split(' ')[1]
       },
-      filename: {
-        sel: 'h3 a',
-        attr: 'title',
-        parse: title => `${title}.jpg`
+      momentDate: {
+        sel: 'div[class="order-label"]',
+        parse: string =>
+          moment(string.match(/\d{1,2}\/\d{2}\/\d{4}/), 'DD-MM-YYYY')
```

Comment:
[`String.prototype.match`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match) returns an array. I don't know how moment handles this: I am astonished it actually works.